ui/app/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -105,8 +108,21 @@ export default function AwaitingSwap({
     category: 'swaps',
   });
 
-  const blockExplorerUrl =
-    txHash && getBlockExplorerUrlForTx({ chainId, hash: txHash }, rpcPrefs);
+  let blockExplorerUrl;
+  if (txHash && rpcPrefs.blockExplorerUrl) {
+    blockExplorerUrl = getBlockExplorerUrlForTx({ hash: txHash }, rpcPrefs);
+  } else if (txHash && SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP[chainId]) {
+    blockExplorerUrl = createCustomExplorerLink(
+      txHash,
+      SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP[chainId],
+    );
+  } else if (txHash) {
```

Comment:
This will use the wrong link for custom networks without a block explorer set.

Not a blocker for this PR, since it's a pre-existing problem, but not ideal. We should at least capture this somewhere so we can fix it later.

Reply:
This is an easy fix and prevents shipping broken links for the test network. So the fix has been added in bcd4b0aee